express/src/app.ts

```diff
@@ -14,6 +15,15 @@ import testingRoutes from "./routes/testing";
 
 const app = Express();
 
+app.use((req, res, next) => {
```

Comment:
These headers are not set in other PR

Reply:
I believe they are: also around line 18 on app.ts?